src/sections/CareersHero/index.css

```diff
@@ -0,0 +1,78 @@
+@import "../../styles/all.css";
+
+.root {
+  padding-bottom: $spacing-224;
+
+  @include Breakpoint-tabletOnly {
+    padding-bottom: $spacing-136;
+  }
+
+  @include Breakpoint-mobileOnly {
+    padding-bottom: $spacing-136;
+  }
+}
+
+.title-desktop {
+  display: flex;
+  flex-direction: column;
+
+  margin-top: calc($spacing-224 - $navbar-height);
+  margin-bottom: $spacing-136;
+
+  @include Breakpoint-tabletAndBelow {
+    display: none;
+  }
+}
+
+.title-mobile {
+  @include Breakpoint-tabletOnly {
+    margin-bottom: $spacing-84;
+  }
+
+  @include Breakpoint-mobileOnly {
+    margin: $spacing-84 0;
```

Comment:
You have too much margin 
![Screenshot 2020-05-06 at 16 04 18](https://user-images.githubusercontent.com/3351677/81193508-6256e000-8fb3-11ea-959f-3bf200c5c4fd.png)
between growth and the wave